### PR TITLE
draco: 1.5.0 -> 1.5.2

### PR DESCRIPTION
**Pull request by jansol**
###### Description of changes

https://github.com/google/draco/releases/tag/1.5.2
https://github.com/google/draco/releases/tag/1.5.1

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/draco/default.nix [hunk 1]
```diff
@@ -11,14 +12,14 @@ let
...
+  version = "1.5.2";
   pname = "draco";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "draco";
     rev = version;
-    hash = "sha256-BoJg2lZBPVVm6Nc0XK8QSISpe+B8tpgRg9PFncN4+fY=";
+    sha256 = "sha256-o6KQJFrnxx2kz6g3vXk1w435OmaJhAJjzsKM/gmVJ2M=";
     fetchSubmodules = true;
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/draco/default.nix:23]: Why not devendor stuff? https://github.com/google/draco/blob/master/BUILDING.md#transcoder

**Comment by nh2** [comment]
This built on ofborg in this PR: https://github.com/NixOS/nixpkgs/pull/165346/checks?check_run_id=5661511449

But the dependency `ghc_filesystem` (which hasn't been changed since) now fails to build on `master`:

```
In file included from /nix/store/l1cds4dpaad47f1n6jwzva2sn1dzdn6a-glibc-2.34-115-dev/include/signal.h:328,
                 from /nix/store/l1cds4dpaad47f1n6jwzva2sn1dzdn6a-glibc-2.34-115-dev/include/sys/param.h:28,
                 from /build/source/include/ghc/filesystem.hpp:51,
                 from /build/source/test/filesystem_test.cpp:62:
/build/source/test/catch.hpp:7860:58: error: call to non-'constexpr' function 'long int sysconf(int)'
 7860 |     constexpr static std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
      |                                                          ^~~~~~~~~~~
In file included from /nix/store/l1cds4dpaad47f1n6jwzva2sn1dzdn6a-glibc-2.34-115-dev/include/bits/sigstksz.h:24,
                 from /nix/store/l1cds4dpaad47f1n6jwzva2sn1dzdn6a-glibc-2.34-115-dev/include/signal.h:328,
                 from /nix/store/l1cds4dpaad47f1n6jwzva2sn1dzdn6a-glibc-2.34-115-dev/include/sys/param.h:28,
                 from /build/source/include/ghc/filesystem.hpp:51,
                 from /build/source/test/filesystem_test.cpp:62:
/nix/store/l1cds4dpaad47f1n6jwzva2sn1dzdn6a-glibc-2.34-115-dev/include/unistd.h:640:17: note: 'long int sysconf(int)' declared here
  640 | extern long int sysconf (int __name) __THROW;
      |                 ^~~~~~~
```

Current Hydra: https://hydra.nixos.org/job/nixpkgs/trunk/ghc_filesystem.x86_64-linux

* Successful Hydra build on 2022-03-16: https://hydra.nixos.org/build/169516619
* Failing Hydra build on 2022-04-05: https://hydra.nixos.org/build/172228296 (cached from https://hydra.nixos.org/build/172075123)

**Comment by jansol** [comment]
I'm guessing a compiler / glibc update broke something.

Looks like upstream has released version 1.5.12 a while ago, might be worth a try to see if that builds ootb.

ping @lourkeur as you are listed as the maintainer for ghc_filesystem and presumably more familiar with that library than I am

**Comment by lourkeur** [comment]
> I'm guessing a compiler / glibc update broke something.
> 
> Looks like upstream has released version 1.5.12 a while ago, might be worth a try to see if that builds ootb.
> 
> ping @lourkeur as you are listed as the maintainer for ghc_filesystem and presumably more familiar with that library than I am

That update is ready in #167301. Can you test with it?

**Comment by nh2** [comment]
> That update is ready in #167301. Can you test with it?

Works: https://github.com/NixOS/nixpkgs/pull/167301#pullrequestreview-942184789

**Pull request merged**